.github/workflows/test.yml

```diff
@@ -82,7 +82,7 @@ jobs:
         name: Running Tests with ${{ matrix.sanitizer }} sanitizer 🚀
         with:
           command: test
-          args: --all-features -Zbuild-std --target x86_64-unknown-linux-gnu
+          args: --all-features -Zbuild-std --target x86_64-unknown-linux-gnu -- --test-threads=1
```

Comment:
setting thread=1 for what?

Reply:
I found it could resolve problem in leak sanitizer test locally, but somehow doesn't work on github workflow.

Reply:
I set `test-threads=1` with reducing concurrency of `ops::oracle::tests::test_detect_conflict`, and passed the leak sanitizer test.